asistente_ladm_col/ui/settings_dialog.ui

```diff
@@ -111,55 +97,73 @@
            </widget>
           </item>
-          <item row="9" column="2">
-           <widget class="QLineEdit" name="txt_pg_password">
-            <property name="echoMode">
-             <enum>QLineEdit::Password</enum>
+          <item row="6" column="3">
+           <widget class="QPushButton" name="create_db_button">
+            <property name="text">
+             <string/>
             </property>
-            <property name="placeholderText">
-             <string>[Leave empty to use system password]</string>
+            <property name="icon">
+             <iconset resource="../resources.qrc">
+              <normaloff>:/Asistente-LADM_COL/resources/images/create_db.png</normaloff>:/Asistente-LADM_COL/resources/images/create_db.png</iconset>
             </property>
            </widget>
           </item>
-          <item row="4" column="2">
-           <widget class="QLineEdit" name="txt_pg_schema">
-            <property name="placeholderText">
-             <string/>
+          <item row="2" column="0">
+           <widget class="QLabel" name="label_5">
+            <property name="text">
+             <string>Password</string>
             </property>
            </widget>
           </item>
-          <item row="3" column="2">
-           <widget class="QLineEdit" name="txt_pg_database">
-            <property name="placeholderText">
-             <string>Database Name</string>
+          <item row="0" column="0">
+           <widget class="QLabel" name="label_4">
+            <property name="text">
+             <string>User</string>
             </property>
            </widget>
           </item>
-          <item row="3" column="0">
+          <item row="6" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Database</string>
             </property>
            </widget>
           </item>
-          <item row="6" column="2">
-           <widget class="QLineEdit" name="txt_pg_user">
+          <item row="2" column="2">
+           <widget class="QLineEdit" name="txt_pg_password">
+            <property name="echoMode">
+             <enum>QLineEdit::Password</enum>
+            </property>
             <property name="placeholderText">
-             <string>Database Username</string>
+             <string>[Leave empty to use system password]</string>
             </property>
            </widget>
           </item>
-          <item row="2" column="2">
-           <widget class="QLineEdit" name="txt_pg_port">
-            <property name="placeholderText">
-             <string>[Leave empty to use standard port 5432]</string>
+          <item row="4" column="0">
+           <widget class="QLabel" name="label_9">
+            <property name="text">
+             <string>Host</string>
+            </property>
+           </widget>
+          </item>
+          <item row="7" column="2">
+           <widget class="QComboBox" name="selected_schema_combobox"/>
```

Comment:
There is a convention to name controls, please use it.
Suggested: "cmb_schema" 
Prefix: Widget type
Suffix: Name